cgmes/cgmes-conversion/pom.xml

```diff
@@ -136,14 +136,6 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
-
-        <!-- Test with all known triple store engines -->
-        <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>powsybl-triple-store-impl-jena</artifactId>
```

Comment:
Not sure you remove this on purpose in this PR?

Reply:
Jena library seems to include an old version of xerces that made the fix impossible: the properties to disable access to external entities in XML parsing were introduced in a more recent version